### ISSUE TITLE
Request for updating version and dependencies of PyPI PixivPy package

### ISSUE THREAD

**Issue by myl7** [upbit/pixivpy#108]
`PixivPy` package in PyPI works almost the same as the latest released version 3.5.4 one in the repo but marked as version 3.5.2.

And as version 3.5.4 added `requests-toolbelt` to dependencies, the package in PyPI does not have this dependencies, which causes ModuleNotFoundError for AppPixivAPI.

Reproduce:

```console
$ python -V && pip -V
Python 3.8.0
pip 19.2.3 from ... (python 3.8)
$ python -m venv venv
$ source venv/bin/activate
$ pip install pixivpy
$ python -c 'from pixivpy3 import AppPixivAPI'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File ".../venv/lib/python3.8/site-packages/pixivpy3/__init__.py", line 8, in <module>
    from .bapi import ByPassSniApi
  File ".../venv/lib/python3.8/site-packages/pixivpy3/bapi.py", line 4, in <module>
    from requests_toolbelt.adapters import host_header_ssl
ModuleNotFoundError: No module named 'requests_toolbelt'
```

**Comment by upbit** [comment]
En, need to add `requests-toolbelt` to the dependency library.

Thanks for feedback.

**Comment by upbit** [comment]
Fixed in v3.5.5

```bash
$ pip uninstall requests-toolbelt
Uninstalling requests-toolbelt-0.9.1:
  Would remove:
    /anaconda2/lib/python2.7/site-packages/requests_toolbelt-0.9.1.dist-info/*
    /anaconda2/lib/python2.7/site-packages/requests_toolbelt/*
Proceed (y/n)? y
  Successfully uninstalled requests-toolbelt-0.9.1
$ pip install pixivpy
Collecting pixivpy
  Downloading https://files.pythonhosted.org/packages/a6/d4/d15ce47d66b17dd81fd5af830f07267c89c1234c207d0180a4e7f292cbbb/PixivPy-3.5.5-py2-none-any.whl
Requirement already satisfied: requests>=2.0 in /anaconda2/lib/python2.7/site-packages (from pixivpy) (2.22.0)
Collecting requests-toolbelt>=0.9.1 (from pixivpy)
  Using cached https://files.pythonhosted.org/packages/60/ef/7681134338fc097acef8d9b2f8abe0458e4d87559c689a8c306d0957ece5/requests_toolbelt-0.9.1-py2.py3-none-any.whl
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /anaconda2/lib/python2.7/site-packages (from requests>=2.0->pixivpy) (3.0.4)
Requirement already satisfied: idna<2.9,>=2.5 in /anaconda2/lib/python2.7/site-packages (from requests>=2.0->pixivpy) (2.8)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /anaconda2/lib/python2.7/site-packages (from requests>=2.0->pixivpy) (1.24.1)
Requirement already satisfied: certifi>=2017.4.17 in /anaconda2/lib/python2.7/site-packages (from requests>=2.0->pixivpy) (2018.11.29)
Installing collected packages: requests-toolbelt, pixivpy
Successfully installed pixivpy-3.5.5 requests-toolbelt-0.9.1
```
